assembl/static2/js/app/components/debate/common/ideas.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Translate } from 'react-redux-i18n';
+import { Grid } from 'react-bootstrap';
+import _filter from 'lodash/filter';
+import IdeasLevel from './ideasLevel';
+
+class Ideas extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
Same comment here, can we use the store instead of the state and create a pure component?